js/layers_fn.js

```diff
@@ -1,151 +1,150 @@
-
```

Comment:
@lynnnakiganda ,

Changes in this file aren't clear. Please see if you can paste back the origin file and then add any change that you're supposed to add. Do this in a commit and push to the same branch for this PR.

Reply:
okay

Reply:
i can't change the same thing in this branch because its seeing like nothing changed